options/cf_options.cc

```diff
@@ -748,11 +748,6 @@ static std::unordered_map<std::string, OptionTypeInfo>
         {"max_mem_compaction_level",
          {0, OptionType::kInt, OptionVerificationType::kDeprecated,
           OptionTypeFlags::kNone}},
-        {"max_write_buffer_number_to_maintain",
```

Comment:
I think we usually keep the option here but mark it as OptionVerificationType::kDeprecated

Reply:
Thanks, that makes sense. Let me add it back.